compiler/src/org.graalvm.compiler.replacements.amd64/src/org/graalvm/compiler/replacements/amd64/AMD64ArrayIndexOf.java

```diff
@@ -106,10 +99,6 @@ public static int indexOf4Chars(char[] array, int arrayLength, int fromIndex, ch
         return AMD64ArrayIndexOfDispatchNode.indexOf(STUB_INDEX_OF_4_CHARS, array, arrayLength, fromIndex, c1, c2, c3, c4);
     }
 
-    public static int indexOf1Char(byte[] array, int arrayLength, int fromIndex, char c) {
-        return AMD64ArrayIndexOfDispatchNode.indexOf(STUB_INDEX_OF_1_CHAR_COMPACT, array, arrayLength, fromIndex, c);
-    }
-
     public static int indexOf2Chars(byte[] array, int arrayLength, int fromIndex, char c1, char c2) {
```

Comment:
I liked that you called this indexOf1CharCompact. Can you also change indexOf(2|3|4)Chars to have the Compact suffix?